server/lib/pbench/__init__.py

```diff
@@ -232,7 +233,7 @@ def __init__(self, cfg_name):
                 " TO-LINK" \
                 " TO-INDEX INDEXED WONT-INDEX" \
                 " TO-COPY-SOS COPIED-SOS" \
-                " TO-BACKUP" \
+                " TO-BACKUP BACKED-UP BACKED-UP-LOCAL BACKED-UP-S3 BACKED-UP-FAILED" \
```

Comment:
Do we need all these? We do need a `TO-BACKUP-LARGE` to squirrel away large objects for now, but I think only `TO-BACKUP TO-BACKUP-KALRGE BACKED-UP` are needed.

Reply:
Good question...